pinc/project_quick_check.inc

```diff
@@ -466,17 +471,9 @@ function _test_project_for_bad_bytes($projectid)
 
         // write image and round columns
         $rowspan = $number_bad_chars;
-        $img_url = "$code_url/tools/page_browser.php?project=$projectid&imagefile=$imagename";
-        $text_url = "$code_url/tools/project_manager/downloadproofed.php?project=$projectid&amp;image=$imagename&amp;round_num=$round_num";
+        $page_browser_url = "$code_url/tools/page_browser.php?project=$projectid&imagefile=$imagename&amp;round_id=$round_name&mode=imageText";
         $details_for_this_page = "<tr>\n";
-        $details_for_this_page .= "<td class='top-align' rowspan='$rowspan'><a href='$img_url'>$imagename</a></td>\n";
```

Comment:
should we leave the img link with the `mode=image` to not break anyone's habits?

Reply:
I defer to @srjfoo on what she thinks is best. My thought was that having a single view of them together is more useful and the reason they were separate was because we _couldn't_ put them together.

Reply:
I saw that and squinted at it a little bit before I understood, and then thought "what a good idea, would that work in the page details?" -- but upon investigation, decided that the page details table is probably better off with two separate columns.

I'm good with combining them into one column in Project Quick Check.

Reply:
Would it be to weird to show Image only if clicking on the Image link (you wouldn't know what text to show anyway - you'd have to show OCR, but maybe not what the user wants), but Text & Image if clicking on the Text link? It feels like showing the image for free when they want to see the text might be helpful and at worst would be irrelevant. Very happy with it

Reply:
Or maybe it's not "for free" - on my wide screen with vertical layout, it is for free, but on a narrower screen or with horizontal layout, the image takes up screen space which could be annoying if you didn't want it.

Reply:
> Would it be to weird to show Image only if clicking on the Image link ...

My comment above applies only to the Project Quick Check change where @cpeel combined the png number and the round into the same column, and the link brings up both image and text for that round.

Reply:
Sorry, I didn't make it clear I was commenting on your "what a good idea, would that work in the page details?"

Reply:
> Sorry, I didn't make it clear I was commenting on your "what a good idea, would that work in the page details?"

Ah, okay. And apparently I didn't make it clear enough that I decided that it wasn't such a good idea after all 😁

Reply:
You were clear - it just made me think 😁 - definitely leave it as it is.